src/controllers/GitHubController.ts

```diff
@@ -0,0 +1,74 @@
+import ProjectData, { ProjectMember } from '../models/ProjectData';
+import * as Octokit from "@octokit/rest";
+
+type RepoType = "public" | "private" | "forks" | "all";
+
+export default class GitHubControler {
+    org: string;
+    type: RepoType;
+    octokit: Octokit;
+
+    constructor(authToken: string, org?: string, type?: RepoType) {
+        this.org = org || "aggie-coding-club";
+        this.type = type || "public";
+        
+        this.octokit = new Octokit({auth: authToken});
+    }
+    
+    getProjectData(): Promise<ProjectData[]> {
+        return this.octokit.repos
+            .listForOrg({
+                org: this.org,
+                type: this.type
+            })
+            .then((reposResponse) => {
+                const repos = reposResponse.data;
+                let projectList: ProjectData[] = [];
+
+                for (let repo of repos) {
+                    let project: ProjectData = {
```

Comment:
You're not modifying the `project` variable, it's best to turn it `const`.

```suggestion
                    const project: ProjectData = {
```